src/test/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/MessageTrailerTest.java

```diff
@@ -4,7 +4,6 @@
 import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
```

Comment:
Should we remove the public identifier for this class?

Reply:
What do you mean?

Reply:
What do you mean by that?

Reply:
Sorry, I could not select that exact line but basically the 

public class MessageTrailerTest {

can be 

class MessageTrailerTest {

If you agree of course

Reply:
JUnit 5 does not need test classes and methods to be public, and idiomatic use of the tool has led the community not to use them. It follows the principle of reducing elements' visibility as much as possible, and also helps the reader to immediately identify that it is indeed a test class using JUnit 5, and not an earlier version where test elements had to be public.